opentelemetry-java-instrumentation.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: opentelemetry-java-instrumentation
+  version: "2.16.0"
+  epoch: 0
+  description: OpenTelemetry auto-instrumentation and instrumentation libraries for Java
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - busybox
+  target-architecture:
+    - x86_64
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - openjdk-21-default-jdk
+  environment:
+    JAVA_HOME: /usr/lib/jvm/java-21-openjdk
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/open-telemetry/opentelemetry-java-instrumentation
+      tag: v${{package.version}}
+      expected-commit: d53c0e95b6ba6fdc62d8cda0c370f9189838f932
+
+  - runs: |
+      ./gradlew -PskipTests=true :javaagent:shadowJar
```

Comment:
why we didn't run `assemble`? https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/CONTRIBUTING.md#building-from-source